ChangeLog

```diff
@@ -1,3 +1,23 @@
+2019-05-23  randomee  <randomee@users.noreply.github.com>
```

Comment:
We would really prefer a real name and a resolvable email address here.

Reply:
Honestly, I'd prefer not to put that info in public.  Past bad on-line to off-line issues.  I'm happy to give you info backchannel, or meet in the loop next time I'm downtown.

Reply:
While I (nominally) understand where you are coming from, allow me to state that we are the (very public) face of this code, and have been for a decade (or longer if you count predecessor).  People will come to John, Whit, myself or now Alfred because there is an identifiable counterparty as well as some trust and visibility.   

I simply have a harder time standing behind code from a random unidentified strangers.  So as (current) Maintainer, I may be inclined to say no to this pull request for this reason.   At the end of the day I simply have no idea who you are, or where you are from.  We will discuss, but it would also be good if you could give this some thought over the long weekend, or if need be, the next few weeks.